src/ontology/cl-edit.owl

```diff
@@ -28114,6 +28113,35 @@ AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030018 "2022-06-27T1
 AnnotationAssertion(rdfs:label obo:CL_4030018 "kidney connecting tubule principal cell")
 EquivalentClasses(obo:CL_4030018 ObjectIntersectionOf(obo:CL_0005009 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005097)))
 
+# Class: obo:CL_4030019 (kidney connecting tubule intercalated cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030019 "An epithelial cell that is part of the connecting tubule with important roles in the regulation of acid-base homeostasis."^^xsd:string)
```

Comment:
your equivalent class suggests that your definition should be: 
"A [renal intercalated cell](http://purl.obolibrary.org/obo/CL_0005010) that is part of the [renal connecting tubule](http://purl.obolibrary.org/obo/UBERON_0005097)."
The acid-base regulation part can be the gloss